base/include/PerspectiveTransform.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include "FrameMetadata.h"
+#include "Module.h"
+
+class PerspectiveTransformProps : public ModuleProps
+{
+public:
+    PerspectiveTransformProps(const std::vector<cv::Point2f> &_srcPoints, const std::vector<cv::Point2f> &_dstPoints)
+    {
+        srcPoints = _srcPoints;
+        dstPoints = _dstPoints;
+    }
+    std::vector<cv::Point2f> srcPoints;
```

Comment:
This should be the basic mode. We should add a mode that takes the source and destination points as input pins in a pipeline so that this module can be used dynamically in a pipeline. Mode=DYNAMIC